drivers/serial/Kconfig.ns16550

```diff
@@ -66,6 +84,14 @@ config UART_NS16550_ACCESS_IOPORT
 	  When enabled, NS16550 will not be a memory mapped device. This option
 	  must be selected at SoC/board level if needed.
 
+config UART_NS16550_SIMULT_ACCESS
+	bool
+	default n
+	help
+	  When enabled, NS16550 supports IO, MMIO, PCIe uart devices simultaneously.
```

Comment:
Here you're spelling it uppercase NS16550, which seems correct. But: UART (not "uart").